nss-day-cohort-46/holiday-road-unnamed-riddle
# Dr park weather

# Description

-This branch integrates the weatherList.js with parkPreview.js so that the weather is displayed on the dom with the park preview. 
-Includes new event.detail on parkSelected so that the name of the park is broadcast.
-Includes minor css so the image is smaller in park preview.


## Type of change

Please delete options that are not relevant.



# Testing Instructions

```
git fetch —all
git checkout dr-park-weather
serve
```
Weather should display along with park detail when a park is selected.

# Checklist:
